### DISCUSSION TITLE
刚升级到0.7.1版本，就无法添加音频文件了，看截图

### DISCUSSION THREAD

**Discussion by thinkbankcoin** [ZuodaoTech/everyone-can-use-english#1193] [General]
看一下截图，在添加音频文件时，文件全部为灰色，无法被选中
![Uploading 截屏2024-11-20 20.27.06.png…]()

**Comment by thinkbankcoin** [comment]
过了20分钟后，好像又可以了

**Comment by thinkbankcoin** [comment]
好像还是不行，如果在Home那里选择添加音频是不可以的。
但在Audios那是选择添加是可以的。

**Comment by an-lee** [comment]
Home 页面上有 `Added Audios` 和 `Added Videos`，两个添加按钮，一个是添加音频，一个是添加视频。
